src/components/Router.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```

Comment:
Do we want to change all these files to be .tsx instead of .js since we are using typescript? If you change the router file to .tsx then you will also need to run npm install --save @types/react-router-dom to get rid of the errors.